core/templates/pages/contributor-dashboard-page/contributor-dashboard-page.component.ts

```diff
@@ -91,7 +93,20 @@ angular.module('oppia').component('contributorDashboardPage', {
       };
       ctrl.onTabClick = function(activeTabName) {
         ctrl.activeTabName = activeTabName;
+        if (ctrl.activeTabName === 'translateTextTab') {
+          setTimeout(() => {
```

Comment:
Can you add a comment explaining why this is needed?

Reply:
Ditto

Reply:
done.